backend/models/AutoCounter.model.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require('mongoose')
+
+const Schema = mongoose.Schema
+
+const AutoCounterSchema = new Schema({
+    _id: { type: String, required: true },
+    seq: { type: Number, default: 0 },
+})
+
```

Comment:
Instead of defining our own AutoCounter, let's use https://www.npmjs.com/package/mongoose-auto-increment.
Make sure to also specify the field to be 'ticket_id' to replace their default field name of '_id' (https://www.npmjs.com/package/mongoose-auto-increment#want-a-field-other-than-_id-)